### PR TITLE
feat: add fetch acl sample code for content warehouse

### PR DESCRIPTION
**Pull request by paulinawins**
## Description
Fixes issue

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [x] These samples need a new **API enabled** in testing projects to pass (let us know which ones). content-warehouse API
- [x] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [x] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [x] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by paulinawins** [comment]
Hi @telpirion or @holtskinner, could either of you review this PR (adds a nodeJS sample code for document warehouse)?

File: .github/blunderbuss.yml [hunk 1]
```diff
@@ -58,3 +63,8 @@ assign_prs_by:
   - 'api: dlp'
   to:
   - GoogleCloudPlatform/googleapis-dlp
+- labels:
+    - 'api: contentwarehouse'
+    - 'api: documentai'
+  to:
+    - GoogleCloudPlatform/document-ai-samples-contributors
```
Comment (kweinmeister) [inline .github/blunderbuss.yml:70]: Nit: keep final new line

File: CODEOWNERS [hunk 2]
```diff
@@ -58,6 +58,7 @@ dialogflow @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/nodejs-samples-
 dialogflow-cx @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/nodejs-samples-reviewers
 discoveryengine @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/nodejs-samples-reviewers
 document-ai @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/nodejs-samples-reviewers
+document-warehouse @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/nodejs-samples-reviewers
```
Comment (kweinmeister) [inline CODEOWNERS:61]: Can we also include `document-ai-samples-contributors` here? Can be handled in another PR if you'd like.

File: document-warehouse/test/fetch-acl.test.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const cwd = path.join(__dirname, '..');
+const PROJECT_ID_PASS = 'YOUR_PROJECT_ID';
+const PROJECT_ID_FAILED = 'PROJECT_ID_WITHOUT_ACL';
+const LOCATION = 'us';
+const USER_ID = 'user:xxxx@example.com';
+const DOCUMENT_ID = 'YOUR_DOCUMENT_ID';
+
+//TODO: Create a PR for this! :)
```
Comment (kweinmeister) [inline document-warehouse/test/fetch-acl.test.js:30]: Remove this comment?

**Review by kweinmeister — CHANGES_REQUESTED** [review]
Looks good, but minor changes requested. Thank you!

**Comment by kweinmeister** [comment]
Also if you can address these header check issues that would be great. The conventional commit check is not required, and not critical to me to change at this point (but other Node reviewers please feel free to assert this).

```
document-warehouse/.eslintrc.yml is missing a valid license header.
document-warehouse/test/.eslintrc.yml is missing a valid license header.
document-warehouse/test/fetch-acl.test.js is missing a valid copyright line.
```

File: document-warehouse/test/.eslintrc.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
```
[the remark this answers is not in the document]
Comment (paulinawins) [reply document-warehouse/test/.eslintrc.yml:1]: One is intended only for the tests in this directory and the other is for the sample code

**Review by paulinawins — COMMENTED** [review]
Addressed all of these requests

**Review by paulinawins — COMMENTED** [review]
marking conversations as resolved

**Review by kweinmeister — APPROVED** [review]

**Review by pattishin — APPROVED** [review]

**Pull request merged**